### PR TITLE
mm member photo

### LINKED ISSUE #112: Add member image for Meryl

### PR DESCRIPTION
**Pull request by ctc-devops**
closes #112

### CODE REVIEW

File: website/src/app/views/about/members.js [hunk 1]
```diff
@@ -316,6 +317,13 @@ const members = {
       linkedinURL: 'https://www.linkedin.com/in/maximalayev/',
       imageURL: maxim,
     },
+
```
Comment (Vakhshoori101) [inline website/src/app/views/about/members.js:320]: Remove this empty line pls

**Review by Vakhshoori101 — COMMENTED** [review]
After removing those empty lines, pls fix the merge conflict then it should be ready to merge!

File: website/src/app/views/about/members.js [hunk 2]
```diff
@@ -334,6 +342,7 @@ const members = {
       linkedinURL: 'https://www.linkedin.com/in/michellelin1',
       imageURL: michelleblin,
     },
+
```
Comment (Vakhshoori101) [inline website/src/app/views/about/members.js:345]: remove this empty line pls

File: website/src/app/views/about/members.js [hunk 3]
```diff
@@ -73,6 +74,7 @@ import dang from '../../../images/about/memberImages/dang.jpg';
 import rachel from '../../../images/about/memberImages/rachel.jpg';
 import joshuamontellano from '../../../images/about/memberImages/joshuamontellano.jpg';
 
+
```
Comment (Vakhshoori101) [inline website/src/app/views/about/members.js:77]: pls remove this empty line pls :) Also do one more merge conflict fix

**Review by ZubairSidhu — APPROVED** [review]

**Pull request merged**